app/controllers/avo/actions_controller.rb

```diff
@@ -16,7 +16,6 @@ def show
       @view = :new
 
       @resource.hydrate(model: @model, view: @view, user: _current_user, params: params)
-      @model = ActionModel.new @action.get_attributes_for_action
```

Comment:
Not sure that this is the right decision.
I can't remember why, but at some point I know for a fact that it wasn't the right decision to use the `@model`.
Let's talk about it.